tests/unit/repo/test_projects.py

```diff
@@ -16,17 +16,18 @@
 
 import pydantic
 import pytest
-from craft_archives.repo.projects import AptDeb, AptPPA
+from craft_archives.repo import errors
+from craft_archives.repo.projects import Apt, AptDeb, AptPPA
 
 
 class TestAptPPAValidation:
     """AptPPA field validation."""
 
-    def test_apt_ppa_valid(self):
-        repo = {
-            "type": "apt",
-            "ppa": "test/somerepo",
-        }
+    @pytest.mark.parametrize(
+        "priority", ["always", "prefer", "defer", 1000, 990, 500, 100, -1, None]
+    )
+    def test_apt_ppa_valid(self, priority):
+        repo = {"type": "apt", "ppa": "test/somerepo", "priority": priority}
```

Comment:
For symmetry with `test_apt_deb_valid()`, how about if priority is `None` we don't add it to the `repo` dict (as opposed to adding a key with `None` value)?